lib/sles4sap_publiccloud.pm

```diff
@@ -621,12 +621,13 @@ sub create_playbook_section_list {
     my ($ha_enabled) = @_;
     my @playbook_list;
 
-    # Add registration module as first element - "QESAP_SCC_NO_REGISTER" skips scc registration via ansible
-    push @playbook_list, 'registration.yaml -e reg_code=' . get_required_var('SCC_REGCODE_SLES4SAP') . " -e email_address=''"
-      unless (get_var('QESAP_SCC_NO_REGISTER'));
-
-    # Add "fully patch system" module after registration module and before test start/configuration moudles
-    push @playbook_list, 'fully-patch-system.yaml';
+    # "QESAP_SCC_NO_REGISTER" skips scc registration via ansible
+    unless (get_var('QESAP_SCC_NO_REGISTER')) {
+        # Add registration module as first element
+        push @playbook_list, 'registration.yaml -e reg_code=' . get_required_var('SCC_REGCODE_SLES4SAP') . " -e email_address=''";
+        # Add "fully patch system" module after registration module and before test start/configuration moudles
+        push @playbook_list, 'fully-patch-system.yaml'
```

Comment:
I'm including this one in its own PR https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/17895